gym/spaces/dict.py

```diff
@@ -21,7 +21,7 @@ class Dict(Space):
             )),
             'rear_cam': spaces.Box(low=0, high=1, shape=(10, 10, 3)),
         }),
-        'ext_controller': spaces.MultiDiscrete((5, 2, 2)),
+        'ext_controller': spaces.MultiDiscrete([ [0,4], [0,1], [0,1] ]),
```

Comment:
I think this needs to be spaces.MultiDiscrete((5, 2, 2)) is the correct version, the other one would not work